dev-vcs/git-lfs/git-lfs-2.13.3.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+EGO_PN=github.com/git-lfs/git-lfs
+inherit go-module
+
+DESCRIPTION="Command line extension and specification for managing large files with git"
+HOMEPAGE="https://git-lfs.github.com/"
+
+if [[ "${PV}" = 9999* ]]; then
+	EGIT_REPO_URI="https://${EGO_PN}"
+	inherit git-r3
+else
+	SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="Apache-2.0 BSD BSD-2 BSD-4 ISC MIT"
+SLOT="0"
+IUSE="doc test"
```

Comment:
Your `doc` IUSE should be `man` instead, since it only installs bunch of manpages.